src/App.tsx

```diff
@@ -20,17 +26,132 @@ class App extends React.Component<{}, State> {
       './img/9.png',
       './img/10.png',
     ],
+    itemWidth: 130,
+    frameSize: 3,
+    step: 3,
+    animationDuration: 1000,
+    isInfinite: false,
+  };
+
+  changeItemWidth = (value: number) => {
+    this.setState({ itemWidth: value });
+  };
+
+  changeFrameSize = (value: number) => {
+    this.setState({ frameSize: value });
+  };
+
+  changeStep = (value: number) => {
+    this.setState({ step: value });
+  };
+
+  changeAnimationDuration = (value: number) => {
+    this.setState({ animationDuration: value });
+  };
+
+  changeIsInfinite = (value: boolean) => {
+    this.setState({ isInfinite: value });
   };
 
   render() {
-    const { images } = this.state;
+    const {
+      images,
+      itemWidth,
+      frameSize,
+      step,
+      animationDuration,
+      isInfinite,
+    } = this.state;
 
     return (
       <div className="App">
         {/* eslint-disable-next-line */}
-        <h1>Carousel with {images.length} images</h1>
+        <h1 className="App__title">Carousel with {images.length} images</h1>
```

Comment:
```suggestion
        <h1 className="App__title">
          {`Carousel with ${images.length} images`}
        </h1>
```
I suppose `eslint-disable` is because of your approach of string output